src/Integration/Binding/SolutionBindingOperation.cs

```diff
@@ -116,11 +114,16 @@ public string GetRuleSetFilePath(RuleSetGroup group)
         #endregion
 
         #region Public API
-        public void Initialize()
+        public void Initialize(IEnumerable<Project> projects)
```

Comment:
The projects can't be discovered from the project system anymore; `BindingWorkflow` passes the applicable projects (from the discover step) in.